Aptoide/aptoide-client-v8
# [ASV-2040] AppNext for exclusive

**What does this PR do?**

 This PR aims at implementing ads from AppNext to mature appviews.
To integrate AppNext i also had to increase play services location version to 17.0.0.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppnextBaseConfigurationCustomEvent.java

**How should this be manually tested?**

Run the app check if you see ads on mature apps. If not this could be because of no fill. Check the logs to see if there is any error.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2040](https://aptoide.atlassian.net/browse/ASV-2040)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

Yes, new placements for the new ad units. 


**Code Review Checklist**
